src/createrepo_c.c

```diff
@@ -427,10 +427,10 @@ cr_create_repomd_records_for_groupfile_metadata(const cr_Metadatum *group_metada
 }
 
 /** Creates list of cr_RepomdRecords from list
- *  of additional metadata (cr_Metadatum) 
+ *  of additional metadata (cr_Metadatum)
  *
  * @param additional_metadata       List of cr_Metadatum
- * @param repomd_checksum_type      
```

Comment:
Sorry about this, my editor did it, I can get rid of these changes if you want.

Reply:
I think its not a problem here.